mava/systems/jax/system.py

```diff
@@ -113,7 +119,14 @@ def add(self, component: Any) -> None:
                 self.config.add(**config_feed)
 
     def build(self, **kwargs: Any) -> None:
-        """Configure system hyperparameters."""
+        """Build the system with all components and hyperparameters.
+
+        Args:
+            **kwargs: dictionary {config name: value} to replace system config values.
+
+        Returns:
+            None.
```

Comment:
If `build` still has the `None` return type in the comment then maybe it should be added to `update`, `add` and `launch` for consistency?

Reply:
Agreed, done!